recipes/stlab/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+import os
+
+class Stlab(ConanFile):
+    name = 'stlab'
+    description = 'The Software Technology Lab libraries.'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/stlab/libraries'
+    license = 'BSL-1.0'
+    topics = 'conan', 'c++', 'concurrency', 'futures', 'channels'
+
+    settings = 'compiler'
+
+    no_copy_source = True
+    _source_subfolder = 'source_subfolder'
+
+    requires = 'boost/1.69.0'
```

Comment:
Why not Boost 1.72.0 ?

Reply:
Good question. It's the version required by the library.

Reply:
Can we try 1.72.0 instead? I think it would nice.

Reply:
What do you think @FelixPetriconi?